### ISSUE TITLE
Do any relations imply another relation?

### ISSUE THREAD

**Issue by ejin700** [hetio/hetionet#52]
Do any of the relations in Hetionet imply the existence of another relation (that isn't the reverse relation)? For instance, there is a Disease - upregulates - Gene edge and a Disease - associates - Gene edge. In this case, does the Disease - upregulates - Gene edge imply that there should also be a Disease - associates - Gene edge?

From the documentation, I see that the data sources for the associates and upregulates edges are different, but from a biological perspective, if a disease upregulates a gene, does that mean it's also classified as being associated with that gene?

**Comment by dhimmel** [comment]
> Do any of the relations in Hetionet imply the existence of another relation

No, although some are actually mutually exclusive like _Compound–palliates–Disease_ and _Compound–treats–Disease_.

The best starting point for details on edges and their sources is the [methods section here](https://git.dhimmel.com/rephetio-manuscript/#edges).

> from a biological perspective, if a disease upregulates a gene, does that mean it's also classified as being associated

_Disease–upregulates–Gene_ edges are pretty noisy so I'd be hesitant to dilute the signal in _association_ edges with these more high-throughput and questionable edges. Users could decide to group edge types, but from a network construction perspective we chose to keep them separate.
